pkg/cache/cache_test.go

```diff
@@ -736,29 +731,42 @@ func TestCacheWorkloadOperations(t *testing.T) {
 					UsedResources: Resources{"cpu": {"on-demand": 0, "spot": 0}},
 				},
 				"two": {
-					Workloads:     sets.NewString("a", "b", "d", "e"),
+					Workloads:     sets.NewString("a", "d", "e"),
 					UsedResources: Resources{"cpu": {"on-demand": 10, "spot": 15}},
 				},
 			},
 		},
 		{
 			name: "delete",
 			operation: func() error {
-				w := kueue.Workload{
-					ObjectMeta: metav1.ObjectMeta{Name: "a"},
-					Spec: kueue.WorkloadSpec{
-						Admission: &kueue.Admission{ClusterQueue: "two"},
+				wls := []kueue.Workload{
```

Comment:
Change to `MakeWorkload` in a follow up PR for we have a fair amount of codes to change.

Reply:
This has been addressed here https://github.com/kubernetes-sigs/kueue/pull/262, prefer this pr merged first.